src/archiefvernietigingscomponent/report/tests/test_report_creation.py

```diff
@@ -147,21 +147,6 @@ def test_report_creation_with_process_owner(self, m):
             "Declaration of destruction - Winter cases (2021-05-05)", report.title
         )
 
-        report.content.seek(0)
```

Comment:
In `test_utils.py` there are the functions that test the content of the report (before it is turned into a PDF). 
Before, when the content was HTML, I used to test for the content also in this test. This was a bit of a duplicate test, but now I can't do this anyway because the report returned is a PDF, so the content is binary. 
This is why I removed this part of the test.